### PR TITLE
bug report template for migrations

### PR DESCRIPTION
**Pull request by girishso**
### Summary

I had to create this for testing migrations in isolation.  Thought it will be helpful in future, without having to dig through the migrations tests.

### CODE REVIEW

**Comment by eileencodes** [comment]
Hey @girishso thanks for the PR! Our other bug report templates have a version for the gem and a version for the master branch. Do you mind updating your PR to include both of those?

**Comment by prathamesh-sonpatki** [comment]
If we are going to add then we need to mention it [here](http://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#create-an-executable-test-case) as well.

**Comment by girishso** [comment]
@eileencodes done!

@prathamesh-sonpatki hi there! Done as well!

**Comment by prathamesh-sonpatki** [comment]
@girishso Can you also squash the commits?

File: guides/bug_report_templates/active_record_migrations_master.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+begin
+  require "bundler/inline"
+rescue LoadError => e
+  $stderr.puts "Bundler version 1.10 or later is required. Please update your Bundler"
+  raise e
+end
+
+gemfile(true) do
+  # source "https://rubygems.org"
+  gem "activerecord"
```
Comment (prathamesh-sonpatki) [inline guides/bug_report_templates/active_record_migrations_master.rb]: We should use `gem "rails", github: "rails/rails"` here similar to how it is used in other master bug report templates.

**Review by prathamesh-sonpatki — CHANGES_REQUESTED** [review]

**Comment by sevenseacat** [comment]
squashing commits can be done automatically as part of the merge now :)

File: guides/bug_report_templates/active_record_migrations_master.rb (hunk 1 above)
Comment (girishso) [inline guides/bug_report_templates/active_record_migrations_master.rb]: Done.  Somehow I missed that!

**Review by prathamesh-sonpatki — APPROVED** [review]

**Comment by eileencodes** [comment]
Thanks @girishso! 👍 

(I loveee these new github review features 😍 )

**Pull request merged**